alpha/lib/model/MediaServerNode.php

```diff
@@ -11,6 +11,9 @@ abstract class MediaServerNode extends DeliveryServerNode {
 	const DEFAULT_APPLICATION = 'kLive';
 	const ENTRY_ID_URL_PARAM = 'e';
 	const PARTNER_ID_URL_PARAM = 'p';
+	const EXPLICIT_LIVE_VIEWER_TYPE_URL = 'type';
```

Comment:
to create interface that all inheritors supply URL + user admin + user user please add functions that return it rather then const.